samples/plain.yaml

```diff
@@ -0,0 +1,31 @@
+kind: DevWorkspace
+apiVersion: workspace.devfile.io/v1alpha2
+metadata:
+  name: plain
+spec:
+  started: true
+  routingClass: 'basic'
+  template:
+    projects:
+      - name: web-nodejs-sample
+        git:
+          remotes:
+            origin: "https://github.com/che-samples/web-nodejs-sample.git"
+      - name: devworkspace-operator
+        git:
+          checkoutFrom:
+            remote: amisevsk
+            revision: clone-projects-on-start
```

Comment:
checkout does not seem to work as expected, I got:
```
> git status

On branch main
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   deploy/deployment/kubernetes/objects/devworkspacetemplates.workspace.devfile.io.CustomResourceDefinition.yaml                                                                                                                           
        deleted:    deploy/deployment/openshift/combined.yaml
        deleted:    deploy/deployment/openshift/objects/devworkspace-controller-configmap.ConfigMap.yaml
        deleted:    deploy/deployment/openshift/objects/devworkspace-controller-edit-workspaces.ClusterRole.yaml
        .....
```

Reply:
Should be fixed now.